root/etc/e-smith/templates/etc/shorewall/rules/40services

```diff
@@ -1,8 +1,45 @@
 {
+    # support for old access property
+    sub accessToZones
```

Comment:
The function is used of backward compatibility to avoid problems if some service has not been migrated.